src/components/SummaryList.scss

```diff
@@ -0,0 +1,42 @@
+.SummaryList-header {
+  color: $base-font-color;
+  .SummaryList-header-description {
+    color: $base-font-color;
+    float: left;
+  }
+  .SummaryList-header-value {
+    color: $base-font-color;
+    float: right;
+    clear: right;
+  }
+}
+
+.SummaryList-row {
+  border-top: $border-width-large solid $ts-grey;
+  padding: $su-default 0;
+  list-style-type: none;
+  li {
```

Comment:
I'd like to see this element have a class name and be un-nested from the parent to reduce specificity . `SummaryList-item` or similar.

Reply:
Would a modifier like this work for our bordered instances?
```
SummaryList-item--bordered:not(:last-child) {
  border-bottom: [style];
}
```